bin/beaker-docker

```diff
@@ -2,8 +2,27 @@
 # frozen_string_literal: true
 
 require 'rubygems' unless defined?(Gem)
+require 'beaker'
 require 'beaker-docker'
 
+def dockerfile(hostspec, filename)
+  ENV['BEAKER_HYPERVISOR'] = 'docker'
+  options = Beaker::Options::Parser.new.parse_args(['--hosts', hostspec || '', '--no-provision'])
+  options[:logger] = Beaker::Logger.new(options)
+  network_manager = Beaker::NetworkManager.new(options, options[:logger])
+  network_manager.provision
+  hosts = network_manager.hosts
+
+  if hosts.size != 1
+    options[:logger].error "Found #{hosts.size} hosts, expected 1"
+    exit(1)
+  end
+
+  hypervisor = network_manager.hypervisors['docker']
+  # TODO: private method
+  File.write(filename, hypervisor.send(:dockerfile_for, hosts.first))
```

Comment:
I considered alternatives like writing out multiple files if there are multiple hosts, but I decided to keep it simple for now.